system-configs/.claude/skills/docx/scripts/accept_changes.py

```
@@ -0,0 +1,135 @@
+
+    cmd = [
+        "soffice",
+        "--headless",
+        f"-env:UserInstallation=file://{LIBREOFFICE_PROFILE}",
+        "--norestore",
+        "vnd.sun.star.script:Standard.Module1.AcceptAllTrackedChanges?language=Basic&location=application",
+        str(output_path.absolute()),
+    ]
+
+    try:
+        result = subprocess.run(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)